Tests/Regression/CON4761.php

```diff
@@ -14,9 +14,12 @@
 use ShopwarePlugins\Connect\Components\RandomStringGenerator;
 use Enlight_Components_Db_Adapter_Pdo_Mysql as Pdo;
 use ShopwarePlugins\Connect\Tests\AbstractConnectUnitTest;
+use ShopwarePlugins\Connect\Tests\DatabaseTestCaseTrait;
 
 class CON4761 extends AbstractConnectUnitTest
```

Comment:
Can this testCase can be ranmed to something descriptive?
Maybe add a link to the regression issue in phpdoc class description